backend-docker/server/apiserver.go

```diff
@@ -192,7 +192,7 @@ func (s *Server) inspectVolumeHandler(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	volId := string(dckr.ImageID(vars["volumeID"]))
 
-	imageID := "d755f01f457926e662451aa753a627c84e6e1f76d517d0982000795630673182"
+	imageID := "eudatgef/volume-filelist"
```

Comment:
This imageid is important, please put it in a constant at the beginning of the file